Class/Tools.php

```diff
@@ -652,19 +874,21 @@ public static function curlCall($url, $options = array())
         }
         curl_setopt_array($ch, $curlOptions);
         $response = curl_exec($ch);
+        curl_close($ch);
```

Comment:
Bude opravdu curl_errno fungovat po curl_close?

Reply:
Výborná připomínka. Nevím resp. nedohledal jsem a uschovám to tedy v proměnné.
Btw. mě to přimělo podívat se do dokumentace a dozvídám se, že curl_errno() může v několika případech hodit stejné číslo pro jinou chybu. http://php.net/manual/en/function.curl-errno.php#117721
Tak tu funkci rozšířím ještě o parametr &$error, kterou naplním výsledkem curl_error().